config/x_hh_bbtautau_run2.yaml

```diff
@@ -24,16 +32,1294 @@ processes:
+    type: lnN
+    value: 0.0124
+    processes:
+    - ZH_Hbb_Zll
+    - ZH_Hbb_Zqq
+  - name: pdf_gg
+    type: lnN
+    value: 0.042
+  - name: th_ggH
+    type: lnN
+    processes:
+    - ggH
+    value: 0.039
+  - name: th_qqH
+    type: lnN
+    processes:
+    - qqH
+    value: 0.004
+  - name: pdf_Higgs_ggH
+    type: lnN
+    value: 0.019
+    processes:
+    - ggH
+    - GluGluToRadionToHHTo2B2Tau_M_${MX}
+    - GluGluToBulkGravitonToHHTo2B2Tau_M_${MX}
+  - name: pdf_Higgs_ttH
+    type: lnN
+    value: 0.03
+    processes:
+    - ttH
+  - name: pdf_Higgs_tH
+    type: lnN
+    value: 0.047
+    processes:
+    - tH
+  - name: pdf_Higgs_qqH
+    type: lnN
+    value: 0.021 # 0.1
+    processes:
+    - qqH
+
+  #- name: pdf_Higgs_qqbar
+  #  type: lnN
+  #  value:
+  #  - value: 0.013
+  #    processes:
+  #    - ZH_Hbb_Zll
+  #    - ZH_Hbb_Zqq
+  #  - value: 0.021
+  #    processes:
+  #    - VBFHToTauTau_M125
+  #  - value: 0.017
+  #    processes:
+  #    - WminusHToTauTau_M125
+  #    - WplusHToTauTau_M125
+
+  - name: pdf_DY
+    type: lnN
+    value: 0.007296099203254119
+    processes:
+    - qqH
+
+  - name: QCDscale_VBF
+    type: lnN
+    value: 0.004
+  - name: QCDscale_TT
+    type: lnN
+    value: [0.02458328336,-0.03597553663]
+  - name: QCDscale_DY
+    type: lnN
+    value: 0.02
+  # - name: QCDscale_V
+  #   type: lnN
+  #   value:
+  #   - value: 0.010209719408638887 # 0.008
+  #     processes:
+  #     - W
+  #   - value: 0.0015150713768754273 #0.02
+  #     processes:
+  #     - EWK_WminusToLNu
+  #     - EWK_WplusToLNu
+  #     - EWK_ZTo2L
+  #     - EWK_ZTo2Nu
+  # - name: QCDscale_VH
+  #   type: lnN
+  #   value:
+  #   - value: 0.038
+  #     processes:
+  #     - ZH_Hbb_Zll
+  #     - ZH_Hbb_Zqq
+  #     - ZHToTauTau_M125
+  #   - value: 0.005
+  #     processes:
+  #     - WminusHToTauTau_M125
+  #     - WplusHToTauTau_M125
+  #   - value: 0.016862949452840021
+  #     processes:
+  #     - HZJ_HToWW
+  - name: QCDscale_VV
+    type: lnN
+    value: 0.02509390849009717
+    processes:
+    - VV
+  - name: QCDscale_VVV
+    type: lnN
+    value: 0.0015074766057555197
+    processes:
+    - VVV
+  - name: QCDscale_tW
+    type: lnN
+    value: 0.054
+    processes:
+    - tW
+  - name: QCDscale_singlet
+    type: lnN
+    value: 0.043999532073744727
+    processes:
+    - ST
+  # - name: alpha_s
+  #   type: lnN
+  #   value:
+  #   - value: 0.009 # ok!
+  #     processes:
+  #     - ZH_Hbb_Zll
+  #     - ZH_Hbb_Zqq
+  #     - ZHToTauTau
+  #   - value: 0.026 # ok
+  #     processes:
+  #     - GluGluHToTauTau_M125
+  #   - value: 0.005 # ok
+  #     processes:
+  #     - VBFHToTauTau_M125
+  #   - value: 0.02 # ok
+  #     processes:
+  #     - ttH
+  #   - value: 0.009 # ok!
+  #     processes:
+  #     - WplusHToTauTau
+  #     - WminusHToTauTau
+  #   - value: 0.02518287564
+  #     processes: TT
+
+
+  - name: TT_mass
+    type: lnN
+    processess: TT
+    value: [0.02782108166,-0.02698165247]
```

Comment:
what is this?

Reply:
#### From [here](https://twiki.cern.ch/twiki/bin/view/LHCPhysics/TtbarNNLO#Updated_reference_cross_sections):  There is an uncertainty on the mass <img width="509" alt="image" src="https://github.com/user-attachments/assets/0f25c6e7-18e0-4927-8c63-dc1dc497f5f3">.

### In the same twiki: Recommendations for most analyses
- Quote as total uncertainty the sum in quadrature of the **scale uncertainty** and the **PDF+alphaS uncertainty**.
- For other top-quark property measurements and search analyses, the uncertainty on the top-quark mass can be added in quadrature as well.
- The central values for the cross section, calculated at different sqrt(s) and for several top-quark masses are provided in the tables, together with the different uncertainties. The parametrisations of the cross section as a function of the top-quark mass and sqrt(s) are also given.

Reply:
The floating values are the percentage ones.

Reply:
We don't apply a variation of m_top in this analysis. So, this uncertainty is not relevant to us.